policyengine_us/variables/gov/states/az/tax/income/net_income/az_pension_exclusion.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class az_pension_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Pension Exclusion"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140BOOKLET.pdf#page=18"
+        "https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2021_140BOOKLET.pdf#page=23"
+        "https://www.azleg.gov/ars/43/01022.htm"
+    )
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.az.tax.income.pension_exclusion
+        person = tax_unit.members
+        is_head = person("is_tax_unit_head", period)
+        is_spouse = person("is_tax_unit_spouse", period)
+        pension_income = person("taxable_public_pension_income", period)
```

Comment:
@MaxGhenis would the following qualify as public pension income or are there more variable inputs Yizhou can use?

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/adcfd7fe-76e4-4028-94d2-41ae361fb3cb)